cmd/pulumi-test-language/interface.go

```diff
@@ -274,6 +275,17 @@ type compiledReplacement struct {
 	Replacement string
 }
 
+// programOverrides represent overrides whereby a test may specify a hardcoded or pre-generated program to be used, in
+// place of running GenerateProject on source PCL. This is useful for testing SDK functionality when the requisite
+// program code generation is not yet complete enough to support generating programs which exercise that functionality.
+type programOverride struct {
+	// The name of the test to which this override applies.
+	Test string
+
+	// A path to a directory containing the program to use for the test.
+	Path string
```

Comment:
Does this need to be []string because some tests will change the program, e.g l2-destroy

Reply:
Good catch; updated